content/blog/2016/2016-11-20-what-jvm-versions-are-running-jenkins-the-return.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "What JVM versions are running Jenkins? 2016 Update!"
+tags:
+- general
+- statistics
+author: batmat
+---
+
+CAUTION: **Disclaimer:** What follows contains some opinions or statements which may not be seen as purely factual or neutral. Note that this represents by no mean the general position of the Jenkins governance board. This is solely my opinion as a contributor based on the data I gathered, and what I feel from the feedback of the community at large.
+
+Like for link:https://jenkins.io/blog/2015/11/03/what-jvm-versions-are-running-jenkins/[last year's article about the same subject], yet another link:https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jenkinsci-dev/fo5nKLhZK5U/Zb9jHpBJBQAJ[recent discussion about finally requiring Java 8 to run future versions Jenkins] pushed me to gather some more factual data around it.
+
+TIP: All numbers shown below are derived from the new `link:stats.jenkins.io/plugin-installation-trend/jvms.json[jvms.json]` file now generated automatically every months, after the two related pull-requests  link:https://github.com/jenkins-infra/infra-statistics/pull/21[1] and link:https://github.com/jenkins-infra/infra-statistics/pull/22[2] got merged.
```

Comment:
I would like interested people to have an easy way to check and get involved. I could move that as a footnote to not get in the way of reading the article initially?